source/components/JsxParser.js

```diff
@@ -1,5 +1,6 @@
 import { Parser } from 'acorn-jsx'
 import React, { Component, Fragment } from 'react'
+import { get } from 'lodash'
```

Comment:
:) I won't accept any pull request that introduces a dependency on `lodash`. Though I love the lib, it's huge and bloated, and will massively increase the file-size of `react-jsx-parser`. Can you please refactor to eliminate this dependency?

Reply:
sure! I thought importing partial of the library will only pack functions related to it, but it seems I'm wrong. I'll write it as a seperate helper function and remove reference to lodash.